packages/truffle-config/index.js

```diff
@@ -344,14 +344,17 @@ Config.detect = function(options, filename) {
 };
 
 Config.load = function(file, options) {
-  var config = new Config();
+  var static_config = originalrequire(file);
+
+  var config = static_config.config
+    ? new static_config.config()
```

Comment:
```suggestion
    ? new staticConfig.config()
```

Sorry I'm just playing around with this new button.